src/version.h

```diff
@@ -24,9 +24,15 @@
   #define KIWIX_TOOLS_VERSION "undefined"
 #endif
 
+#include <kiwix/version.h>
+#include <zim/version.h>
+
 void version()
 {
-  std::cout << KIWIX_TOOLS_VERSION << std::endl;
+  std::cout << KIWIX_TOOLS_VERSION << std::endl << std::endl;
```

Comment:
Doesn't it make sense to start the first line printed by this function with "kiwix-tools "?

Reply:
The version is the tool version and the kiwix-tools version. It is the same. But believe printing `kiwix-tools x.y.z` to be more confusing than just `x.y.z`.

We could print something like `kiwix-serve x.y.z` (was the case before I believe) but then the code is a bit more complex. But this is doable.

Current compromise seems to me to be the best compromise.

Reply:
@veloman-yunkan Finally changed my mind ;) Thx for the review!